lib/rmt/lockfile.rb

```diff
@@ -8,7 +8,8 @@ def lock
       File.open(RMT::Lockfile::LOCKFILE_LOCATION, File::RDWR | File::CREAT) do |f|
         if f.flock(File::LOCK_EX | File::LOCK_NB)
           f.write(Process.pid.to_s)
-          f.fsync
+          f.flush
+          f.truncate(f.pos)
```

Comment:
Is it possible to test this?

Reply:
see PR description :slightly_smiling_face:

Reply:
oh if you mean making a spec, I would suggest adding a feature spec for that.

Reply:
added specs :+1: